### PR TITLE
packaging: make working with RHEL8

### PR DESCRIPTION
**Pull request by safchain**
skip skydive-go-fmt
skip skydive-functional-tests-backend-elasticsearch
skip skydive-functional-tests-backend-orientdb
skip skydive-scale-tests
skip skydive-cdd-overview-tests
skip skydive-unit-tests
skip skydive-compile-tests
skip skydive-k8s-tests
skip skydive-ppc64le-tests
skip skydive-functional-tests-ebpf

Remaining warning..

```
(Thu Jan 17 09:44:57:703083 2019) [sss_cache] [confdb_init] (0x0010): Unable to open config database [/var/lib/sss/db/config.ldb]
Could not open available domains
groupadd: sss_cache exited with status 5
groupadd: Failed to flush the sssd cache.
(Thu Jan 17 09:44:57:714083 2019) [sss_cache] [confdb_init] (0x0010): Unable to open config database [/var/lib/sss/db/config.ldb]
Could not open available domains
groupadd: sss_cache exited with status 5
groupadd: Failed to flush the sssd cache.
(Thu Jan 17 09:44:57:762920 2019) [sss_cache] [confdb_init] (0x0010): Unable to open config database [/var/lib/sss/db/config.ldb]
Could not open available domains
useradd: sss_cache exited with status 5
useradd: Failed to flush the sssd cache.
(Thu Jan 17 09:44:57:774363 2019) [sss_cache] [confdb_init] (0x0010): Unable to open config database [/var/lib/sss/db/config.ldb]
Could not open available domains
useradd: sss_cache exited with status 5
useradd: Failed to flush the sssd cache.
ValueError: Port tcp/9200 already defined
```

### CODE REVIEW

**Comment by nplanel** [comment]
confdb_init are related to https://bugzilla.redhat.com/show_bug.cgi?id=1654592

may need to double check (on fresh install) the ValueError: Port tcp/9200 already defined
generated from semanage port ...

**Comment by safchain** [comment]
everything seems to be ok now

File: contrib/packaging/rpm/skydive.spec [hunk 1]
```diff
@@ -49,7 +49,8 @@ BuildRequires:  llvm clang kernel-headers
 %endif
 BuildRequires:  selinux-policy-devel, policycoreutils-devel
 Requires:       %{name}-selinux = %{version}-%{release}
-Requires:       libpcap libxml2 libvirt-libs
+Requires:       libpcap libxml2 libvirt-libs coreutils
+Requires(pre):    /usr/sbin/useradd
```
Comment (lebauce) [inline contrib/packaging/rpm/skydive.spec]: nit: alignment

**Comment by lebauce** [comment]
run skydive-packaging-tests

**Pull request merged**